FlowCryptUI/Nodes/AttachmentNode.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  AttachmentNode.swift
+//  FlowCryptUI
+//
+//  Created by QSD BiH on 16. 4. 2021..
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import AsyncDisplayKit
+
+public struct Attachment {
+    var name, size: NSAttributedString
+}
+
+public final class AttachmentsNode: CellNode {
+    public struct Input {
+        let name: String
+        let size: String
+
+        public init(
+            name: String,
+            size: String
+        ) {
+            self.name = name
+            self.size = size
+        }
+    }
+
+    private var attachmentNodes: [AttachmentNode] = []
+
+    public init(input: [Input]) {
+        super.init()
+        input.forEach { input in
+            attachmentNodes.append(
+                AttachmentNode(
+                    input: AttachmentNode.Input(name: input.name, size: input.size)
+                )
+            )
```

Comment:
instead of using `forEach` you can use `map`
`attachmentNodes = input.map { AttachmentNode.Input(name: $0, size: $0.size) }`

or you can have private extended init for AttachmentNode.Input which can be inited with `Input`
In this case you will have 
`attachmentNodes = input.map(AttachmentNode.Input.init)`